test/transaction.spec.js

```diff
@@ -0,0 +1,50 @@
+var MOCKUP, assert, Transaction;
+
+assert = require('chai').assert;
+Transaction = require('../transaction');
+
+MOCKUP = {
+  'good': {
+    'date': '01/06/2014',
+    'kind': 'normal',
+    'balance': 0,
+    'charge': 15000,
+    'deposit': 0,
+    'description': 'Giro cajero automatico'
+  }
+};
+
+describe('Module Transaction', function () {
+  describe('Set object', function () {
+    it ('Should have valid keys.', function () {
+      var fn = function () {
+        var trans = new Transaction(MOCKUP.good);
+      };
+      assert.doesNotThrow(fn, Error);
+    });
+
+    it ('Should be built.', function () {
+      var fn = function () {
+        var trans = new Transaction(MOCKUP.good);
+        trans.build();
+      };
+      assert.doesNotThrow(fn, Error);
+    });
+
+    it ('should set all property of a new instance and does not throw', function () {
+      var fn = function () {
+        var t = new Transaction();
+
+        t.date('01/06/2014');
+        t.kind('normal');
+        t.balance(0);
+        t.charge(15000);
+        t.deposit(0);
+        t.description('Giro cajero automatico');
+
+        t.build();
+      };
+      assert.doesNotThrow(fn, Error);
```

Comment:
This is not necessary as the test runner will fail the tests when an exception is thrown.

It should be:

```javascript
  var t = new Transaction();

  t.date('01/06/2014');
  t.kind('normal');
  t.balance(0);
  t.charge(15000);
  t.deposit(0);
  t.description('Giro cajero automatico');
  t.build();
```